From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jason Perlow <jperlow@gmail.com>
Date: Wed, 24 Jun 2026 23:33:00 -0400
Subject: [PATCH 62/83] rtc: rx8900: drop removed of_gpio header include

linux/of_gpio.h is removed in v7.1 and this driver does not use any
legacy of_gpio APIs. Drop the stale include to restore compilation.

Signed-off-by: Jason Perlow <jperlow@gmail.com>
---
 drivers/rtc/rtc-rx8900.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/rtc/rtc-rx8900.c b/drivers/rtc/rtc-rx8900.c
index 5edef5563e1a..0c4ad36884da 100755
--- a/drivers/rtc/rtc-rx8900.c
+++ b/drivers/rtc/rtc-rx8900.c
@@ -42,7 +42,6 @@
 #include <linux/i2c.h>
 #include <linux/list.h>
 #include <linux/rtc.h>
-#include <linux/of_gpio.h>
 
 #include <linux/of.h>
 #include <linux/of_device.h>
